src/app/(marketing)/profile/_components/CircularProgressBar.tsx

```diff
@@ -0,0 +1,36 @@
+"use client"
+
+import { ChartContainer } from "@/components/ui/Chart"
+import { RadialBar, RadialBarChart } from "recharts"
```

Comment:
We should use visx (that we already use, we can talk about the reasons tomorrow) or ideally not even that just a simple Progress component with a basic solution (svg morphing maybe, but we should check what's the recommended way to do it)